### PR TITLE
added function to generate info query data, fixed line 101 not choosi…

### PR DESCRIPTION
**Pull request by modernfaust**
Added function to generate info query data (divided by: who, what, where, when, how). 

Also detected & fixed a naming issue on line 101. When running python preprocess.py, certain generated stat queries will sometimes throw exceptions.

### CODE REVIEW

File: preprocess.py [hunk 1]
```diff
@@ -109,14 +109,47 @@ def generate_stat_queries(samples):
...
+            verb = choice(verb_list)
+            query = "what are you,info\n"
+            query1 = "what do you {},info\n".format(verb)
+            query2 = "what can you {},info\n".format(verb)
+            Qs = [query1, query2]
+            query_list.append(choice(Qs))
+        elif randq == 3:
+            query = "who are you,info\n"
+            query1 = "who made you,info\n"
+            Qs = [query, query1, query2]
```
Comment (skekre98) [inline preprocess.py:129]: I'm guessing this is supposed to be `Qs = [query, query1]`?

**Review by skekre98 — CHANGES_REQUESTED** [review]

**Comment by skekre98** [comment]
Just one minor fix, but besides that everything looks great! Out of curiosity, what accuracy is the model receiving with the new training data?

File: preprocess.py (hunk 1 above)
Comment (modernfaust) [reply preprocess.py]: good catch! just pushed the fix

**Comment by modernfaust** [comment]
> 
> 
> Just one minor fix, but besides that everything looks great! Out of curiosity, what accuracy is the model receiving with the new training data?

I ran query_classification a couple of times, ended up with accuracy = 1.0 for both tries. 

I can work on other issues as well :)

**Review by skekre98 — APPROVED** [review]

**Comment by skekre98** [comment]
Awesome! Looks good to me. @modernfaust feel free to take up any issue you'd like ☺️

**Pull request merged**